sql_csv_utils.py

```diff
@@ -106,6 +107,45 @@ def check_agent_name_sql(self, first_name: str, last_name: str, middle_initial:
         else:
             return result
 
+    def check_collector_list(self, collector_list, new_agents=False):
```

Comment:
new function to check for missing or collector unknown tags , and to replace with standard name dict and 'unspecified' tag. Different logic for total agent list vs new agent lists.